docs/userguide/labels-custom-metadata.md

```diff
@@ -59,15 +60,14 @@ notation. Use the following guidelines to name your keys:
   cumbersome namespaces on the command-line.
 
 
-These are guidelines and Docker does not *enforce* them. Failing following these
-guidelines can result in conflicting labels. If you're building a tool that uses
-labels, you *should* use namespaces for your label keys.
+These are simply guidelines and Docker does not *enforce* them. However, for
+the benefit of the community, you *should* use namespaces for your label keys.
 
 
 ## Store structured data in labels
 
-Label values can contain any data type that can be stored as a string. For
-example, consider this JSON:
+Label values can contain any data type as long as it can be represented as a
```

Comment:
@charleswhchan This isn't adding clarity the antecedent `it` is unclear.  Why did you want to change this?